rsts/concepts/admin.rst

```diff
@@ -465,15 +469,15 @@ Only a subset of fields are supported for sorting list queries. The explicit lis
 Sorting syntax	
 --------------	
 
-Adding sorting to a request requires specifying the ``key``, e.g. the attribute you wish to sort on. Sorting can also optionally specify the direction (one of ``ASCENDING`` or ``DESCENDING``) where ``DESCENDING`` is the default.	
+Adding sorting to a request requires specifying the ``key``, ex: the attribute you wish to sort on. Sorting can also optionally specify the direction (one of ``ASCENDING`` or ``DESCENDING``) where ``DESCENDING`` is the default.	
 
 Example sorting HTTP parameter:	
 
 ::	
 
    sort_by.key=created_at&sort_by.direction=DESCENDING	
    	
-Alternatively, since descending is the default, the above could be rewritten as	
+Alternatively, since `DESCENDING` is the default sorting direction, the above could be written as	
```

Comment:
this is rst unf not markdown.  in rst if you want to `monospace` something you have to use two backticks
```
``mono``
```